sky/utils/resources_utils.py

```diff
@@ -0,0 +1,63 @@
+"""Resources related utilities."""
+from sky.clouds import service_catalog
+from sky.utils import ux_utils
+
+# List of all accelerators supported by SkyPilot.
+# NOTE: Must include accelerators supported for local clusters.
```

Comment:
Can you document what happens if:
- a name is in this list, but not in any catalog (can users launch it?)
- a name is not in this list, but in a catalog (can users launch it?)
- for developers, what actions, if any, should we take to keep this list and the catalogs in sync?

Reply:
Added more comments. PTAL.